python/ray/tune/suggest/optuna.py

```diff
@@ -53,6 +59,10 @@ class OptunaSearch(Searcher):
             minimizing or maximizing the metric attribute.
         sampler (optuna.samplers.BaseSampler): Optuna sampler used to
             draw hyperparameter configurations. Defaults to ``TPESampler``.
+        base_config (dict): Base config dict that gets overwritten by the
+            Optuna sampling and is returned to each Tune trial. This could e.g.
+            contain static variables or configurations that should be passed
+            to each trial.
```

Comment:
hmm is there a reason why optuna has a base config?

Reply:
I'll check how we currently do this for bayesopt etc and see if I can keep this for Optuna